ConsoleApp1/ConsoleApp1.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
what is this?

Reply:
It was a redundant project. Now it's removed